### PR TITLE
Feature/implementation parser factory

### PR DESCRIPTION
**Pull request by kogonidze**

### CODE REVIEW

File: Octopus.Core.Parser.WorkerService/Services/ParserProcessor.cs [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+                var request = await _consumer.ConsumeAsync();
+
+                if (request != null)
+                {
+                    var inputFile = new FileInfo(request.EntityFilePath);
+
+                    ParseFile(inputFile);
+                }
+
+                await Task.Delay(1000, stoppingToken);
```
Comment (VladiTitov) [inline Octopus.Core.Parser.WorkerService/Services/ParserProcessor.cs:38]: maybe usinp FluentSheduller fot this? Or you will add this in future features?

Comment (VladiTitov) [inline Octopus.Core.Parser.WorkerService/Services/ParserProcessor.cs:38]: maybe usinp FluentSheduller fot this? Or you will add this in future features?

Comment (kogonidze) [reply Octopus.Core.Parser.WorkerService/Services/ParserProcessor.cs:38]: Sure, it`s temporary measure. I`ll configure FluentScheduler or it`s analogue later

**Comment by VladiTitov** [comment]
Looks good!

**Review by VladiTitov — APPROVED** [review]

**Review by Egorkas — APPROVED** [review]
Bad name for branch))) Watch the name of Vlad's branch. Great job.

**Pull request merged**